src/providers/ecency/ecency.ts

```diff
@@ -116,21 +116,14 @@ export const deleteDraft = async (draftId: string) => {
 };
 
 /**
- * @params title
- * @params body
- * @params tags
- * @param meta
+ * @param draft
  */
-export const addDraft = async (title: string, body: string, tags: string, meta: Object) => {
+export const addDraft = async (draft: Object) => {
+  const { title, body, tags, meta } = draft;
   try {
     const data = { title, body, tags, meta };
     const res = await ecencyApi.post('/private-api/drafts-add', data);
-    const { drafts } = res.data;
-    if (drafts) {
-      return drafts.pop(); //return recently saved last draft in the list
-    } else {
-      throw new Error('No drafts returned in response');
-    }
+    return res.data || [];
```

Comment:
please test addDraft response again, it should return array of drafts... res.data do not contain the expected array....
https://media.discordapp.net/attachments/1094029149892509767/1103977166485598218/Screenshot_2023-05-05_at_2.30.31_PM.png